src/backend/launcher.ts

```diff
@@ -826,6 +829,9 @@ async function callRunner(
     return currentPromise
   }
 
+  const abortId = options?.abortId || appName || Math.random().toString()
```

Comment:
if no abortId is passed, it tries to get the appName from the command, else it generates a random id